cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -101,19 +100,19 @@ elif [[ $realHostname == "waterman"* ]] ; then
   hostnameMatch=waterman
   hostnameMatchSystemName=waterman
 
-# Specifically named cts1 systems (currently maps to 'serrano' env)
+# Specifically named cts1 systems
```

Comment:
@jeforster, please just delete this entire block of code based on 'hostname'.  The 'cts1' env is already detected below by looking at `SNLSYSTEM`.

Reply:
@bartlettroscoe You are correct.  I had forgotten that we were also checking SNLSYSTEM.  I took this part out.